docs/source/developer_guide/metadata.md

```diff
@@ -133,6 +135,8 @@ if not runtime_configuration:
     raise RuntimeError("Runtime metadata not available.")
 
 api_endpoint = runtime_configuration.metadata['api_endpoint']
+api_endpoint_username = runtime_configuration.metadata['api_endpoint_username']
+api_endpoint_password = runtime_configuration.metadata['api_endpoint_password']
```

Comment:
I'm wondering if these should be named simply `api_username` and `api_password`.  This would also be consistent with the naming conventions used for the `cos_` fields.  Since we can't easily change `cos_username` to `cos_endpoint_username`, it would make sense (for consistency's sake) to change the new fields.

Reply:
I like that suggestion and to make them shorter :)